src/cadence/scripts/getNFTIDs.cdc

```diff
@@ -152,5 +152,26 @@ pub fun main(ownerAddress: Address): {String: [UInt64]} {
             ids["Gooberz"] = col.getIDs()
         }
 
+    if let col = owner.getCapability(MintStoreItem.CollectionPublicPath)
```

Comment:
We can remove this section from the script, the code below should work with MintStoreItem.<merchant_name> as the key